_posts/2021-04-26-integrating-airflow-and-okta.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Integrating Airflow with Okta"
+author: kuntalb
+tags:
+- okta
+- airflow
+- featured
+team: Core Platform
+---
+
+
+At Scribd we decided to pair Airflow with Okta. Earlier we were using LDAP for
+authentication. This write up will describe the journey of integrating Airflow
+with Okta from the earlier LDAP setup.
+
+
+## Prerequisite:
+1. Okta with [API Access Management](https://developer.okta.com/docs/concepts/api-access-management/) enabled.
+1. [Flask-AppBuilder
+   3.2.2](https://github.com/dpgaspar/Flask-AppBuilder/tree/v3.2.2). Official
+   Airflow repo has a
+   [constraint](https://github.com/apache/airflow/blob/master/setup.cfg#L97) on
+   `flask-appbuilder~=3.1,>=3.1.1`, so we might need to use a fork to get this
+   integration going.
+1. `sqlalchemy>=1.3.18, <1.4.0`
+1. [ ] `authlib==0.15.3`
```

Comment:
I don't understand these statements as prerequisites, what does a person actually do with these versions?  Some more narrative of how somebody should use this would be helpful